drivers/sensor/hts221/hts221_trigger.c

```diff
@@ -110,8 +133,8 @@ int hts221_init_interrupt(struct device *dev)
 	}
 
 	gpio_pin_configure(drv_data->gpio, DT_INST_0_ST_HTS221_DRDY_GPIOS_PIN,
-			   GPIO_DIR_IN | GPIO_INT | GPIO_INT_EDGE |
-			   GPIO_INT_ACTIVE_HIGH | GPIO_INT_DEBOUNCE);
+			   GPIO_INPUT | GPIO_INT_DEBOUNCE
```

Comment:
The HTS221 should produce high quality signal on its DRDY pin, no ringing, spikes, etc. I believe the `GPIO_INT_DEBOUNCE` is not necessary.

Reply:
Both requested changes made.